src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs

```diff
@@ -0,0 +1,389 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using MSBuild = Microsoft.Build.Utilities;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Threading;
+using System.IO;
+using Microsoft.DotNet.Build.CloudTestTasks;
+using System.Net.Http;
+using Microsoft.Build.Framework;
+using Newtonsoft.Json;
+using NuGet.Versioning;
+using System.Xml;
+using System.Net;
+
+namespace Microsoft.DotNet.Build.Tasks.Feed
+{
+    sealed class BlobFeedAction: IFeedAction<string>
+    {
+        private MSBuild.TaskLoggingHelper Log;
+        private static readonly CancellationTokenSource TokenSource = new CancellationTokenSource();
+        private static readonly CancellationToken CancellationToken = TokenSource.Token;
+        private static readonly string DownloadDirectory = Path.Combine(Directory.GetCurrentDirectory(), "download");
+        private static readonly string IndexDirectory = Path.Combine(Directory.GetCurrentDirectory(), "tmp");
+
+        public BlobFeed feed;
+        public int MaxClients { get; set; } = 8;
+
+        public BlobFeedAction(string accountName, string accountKey, string containerName, MSBuild.TaskLoggingHelper Log)
+        {
+            this.feed = new BlobFeed(accountName, accountKey, containerName, Log);
+            this.Log = Log;
+        }
+
+        public Task<bool> DeleteFromFeed(string filterString = "")
+        {
+            throw new NotImplementedException();
```

Comment:
Why are all these throwing NotImplementedException?

Reply:
I wanted to implement them at some point, should I comment them out or delete them and update the doc to reflect the actual apis needed?